src/site/_data/elements.json

```diff
@@ -0,0 +1,3 @@
+{
+  "button": "node_modules/@visual-framework/vf-button/vf-button.njk"
```

Comment:
What happens if `@visual-framework/vf-button` isn't installed?

Thinking we should add a note to the Readme.md

Reply:
Good point. 

If nobody calls it (`{% include button %}` then it won't do anything. But if they do and it's not installed it won't work.

We could either not include all the shorcodes in the `_data` files or we could and have a note.

From working with Spilios and Filipe with some of this - I think having all the shortcakes setup will be the better option.

Reply:
Yeah, if we could preload most of them that would make life easier. One day it'd be nice to have 11ty auto-magically scan the components and make this list.